config/vendor-functions/lockdown.sh

```diff
@@ -136,7 +136,7 @@ fi
 
 # Output the dm-verity hash
 echo "Hash: ${HASH}"
-read -p "Press enter once you have recorded the system hash."
+read -p "Press enter once you have recorded the system hash. "
```

Comment:
The most minor tweak for stylistic consistency elsewhere in vxsuite-complete-system haha